src/RenderWebGL.js

```diff
@@ -1704,6 +1712,10 @@ class RenderWebGL extends EventEmitter {
         dst[2] += blendAlpha * 255;
         return dst;
     }
+
+    requestSnapshot (callback) {
```

Comment:
Since this is part of the outward-facing API I think it should have a full JSDoc comment.